proto/frequenz/api/weather/weather.proto

```diff
@@ -74,14 +75,12 @@ message ForecastRequest {
   // will be streamed.
   repeated ForecastFeature features = 2;
 
+  // Optional for StreamLiveWeatherForecast.
   // The start of the period for which the forecast is being requested.
-  // If not provided, the service will default to the most recent forecast
-  // available.
   google.protobuf.Timestamp start_ts = 3;
 
+  // Optional for StreamLiveWeatherForecast.
```

Comment:
It would be nice if we could continue specifying what happens if this field is not specified.

Reply:
Same, will the stream stop at that time if this field is provided in the streaming case?

Reply:
> will the stream stop at that time if this field is provided in the streaming case?

No. This should not be applied to the stream.